hos-data-on-live-map/src/scripts/index.js

```diff
@@ -436,11 +433,11 @@ geotab.addin.request = (elt, service) => {
                     .then(function (result) {
                         var multicallresults = result[0];
                         var driverRegulationResult = result[1];
-
-                        var driverDetails = multicallresults[1][0].name + " | " + multicallresults[1][0].hosRuleSet;
```

Comment:
Now that driver details is just hosruleset it would make sense to change the name to hosruleset instead of driver details.